blockchain/backup/NFTtheWorld.sol

```diff
@@ -13,46 +15,66 @@ contract NFTtheWorld {
     uint256[] public dropHashes;
 
     struct NFTOwnership {
-        address owner;
-        uint256 nftId;
+        address payable owner;
+        string uri;
         uint256 dropId;
         uint256 dropTime;
         address reservedFor;
+        uint256 weiPrice;
     }
 
+    // For testing reasons this is public
+    // To check if an address is an admin
+    mapping(address => bool) public adminRights;
```

Comment:
Apropos clear variable names. This variable represents a predicate. Common conventions use `is` or `has` as prefix to allow the variable to explain itself. "What-Comments" mostly can be replaced by self-explaining names like `isAdminAddress` and defining new names. I think, we try to use self-explaining names in the frontend.

It's maybe only me but I find the comments below rather confusing and hindering for understanding.

Reply:
will fix in next issue